src/pages/mypage/index.tsx

```diff
@@ -0,0 +1,34 @@
+import type { NextPage } from "next";
+import Head from "next/head";
+import styled from "@emotion/styled";
+import { PurchaseHistoryCont } from "../../components/PurchaseHistory/CardContainer";
+import { WishListContainer } from "../../components/WishList/CardContainer";
+
+const MyPage: NextPage = () => {
+  return (
+    <section>
+      <Head>
+        <title>마이 페이지</title>
+        <meta name="description" content="Generated by create next app" />
+        <link rel="icon" href="/favicon.ico" />
+      </Head>
+      <MyPageCont>
+        <PageTitle>마이 페이지</PageTitle>
+        <PurchaseHistoryCont />
+        <WishListContainer />
```

Comment:
p3: 폴더명과 동일하게 WishList와 일치되게 WisthList로 해주시는 것이 좋을 것 같습니다. 폴더명과 파일이름이 지금은 전부 다르네요.
```suggestion
        <WishList/>
```

Reply:
이 부분은 제가 이해가 잘 안되는데 나중에 회의 때 말씀해주시면 감사하겠습니다!

Reply:
리액트 컴포넌트 만들떄, 예를 들어 , Button이라는 컴포넌트를 만들면 Button폴더 안에 index.js나 혹은 Button.jsx라고 하잖아요? 하지만 지금 폴더 트리를 보면 WishListContainer라는 폴더가 있는데 폴더명은 cardContainer인가 뭔가 다르게 적혀있더라구요!